client/wfnews-war/src/main/angular/src/app/components/saved/add-saved-location/notificatnio-map/notification-map.component.ts

```diff
@@ -0,0 +1,144 @@
+import { AfterViewInit, ChangeDetectorRef, Component, Inject, OnInit, ViewChild } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import * as L from 'leaflet'
+
+
+@Component({
+  selector: 'wfnews-notificatino-map',
+  templateUrl: './notification-map.component.html',
+  styleUrls: ['./notification-map.component.scss']
+})
+export class notificatinoMapComponent implements OnInit, AfterViewInit  {
```

Comment:
The class name, component selector and folder name have typos here, should match notification-map for consistency

Reply:
Thanks for catching this! silly mistake